charms/trusty/clearwater-aio-proxy/actions/create-user

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+# Get the configuration and action parameters.
+proxied_ip=$(config-get proxied_ip)
+login_password=$(config-get password)
+home_domain=$(config-get home_domain)
+number=$(action-get number)
+password=$(action-get password)
+
+if [ -z "$proxied_ip" ] || [ -z "$login_password" ] || [ -z "$home_domain" ] ; then
+  echo Proxy not yet configured!
+  exit 1
+fi
+
+# Create the user.
+sshpass -p$login_password ssh -o StrictHostKeyChecking=no ubuntu@$proxied_ip "echo $login_password | sudo -S /usr/share/clearwater/bin/create_user $number $home_domain $password"
```

Comment:
I wonder if:
  - we should set our AIO image up with passwordless sudo to simplify this sort of thing
  - we should write these actions in Python instead - this SSH stuff might be slightly cleaner if expressed in a library like Fabric

I don't expect you to make those changes, just musing.